### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by matthewdking**
relates #39 

+ creates nav bar
+ adds nav bar to all pages

### CODE REVIEW

File: src/components/Settings.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  color: palevioletred;
+`;
+
+const SettingsTitle = styled.h1``;
+
+const Settings = () => {
+  return (
+    <SettingsContainer>
+      <NavBar />
+      <SettingsTitle>Hi you're here to change some settings</SettingsTitle>
```
Comment (veredrec) [inline src/components/Settings.js]: It seems like something with the syntax here is wrong, maybe the single quotes makes it weird? Not sure how it looks like in atom.

Comment (matthewdking) [reply src/components/Settings.js]: It looks fine on atom and it seems to render without errors :confused:

Comment (finnhodgkin) [reply src/components/Settings.js]: I guess you could use the html symbol, but it's definitely not a biggie :woman_shrugging:

File: src/containers/CareerDetails.js [hunk 2]
```diff
@@ -10,6 +11,7 @@ class CareerDetails extends Component {
   render() {
     return (
       <CareerContainer>
+        <NavBar />
```
Comment (finnhodgkin) [inline src/containers/CareerDetails.js]: The Navbar is shared between all components on the app so each of these individual navbars should be moved to the `Router`.

Comment (finnhodgkin) [reply src/containers/CareerDetails.js]: Here's a working example of a router:

```js
const Main = () => {
  return (
    <div>
      <NavBar />
      <Switch>
        <Route path="/Careers" exact component={Careers} />
        <Route path="/Careers/Liked" exact component={LikedCareers} />
        <Route path="/Careers/Details/:title" exact component={CareerDetails} />
        <Route path="/Settings" exact component={Settings} />
      </Switch>
    </div>
  );
};

const Routes = () => (
  <BrowserRouter>
    <Switch>
      <Route path="/AddCareer" exact component={AddCareer} />
      <Route path="*" component={Main} />
    </Switch>
  </BrowserRouter>
);
```

File: src/containers/Careers.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import styled from 'styled-components';
+import NavBar from '../components/NavBar';
```
Comment (finnhodgkin) [inline src/containers/Careers.js]: As above

File: src/containers/Careers.js [hunk 4]
```diff
@@ -10,6 +11,7 @@ class Careers extends Component {
   render() {
     return (
       <CareersContainer>
+        <NavBar />
```
Comment (finnhodgkin) [inline src/containers/Careers.js]: And again

File: src/containers/LikedCareers.js [hunk 5]
```diff
@@ -10,6 +11,7 @@ class LikedCareers extends Component {
   render() {
     return (
       <LikedContainer>
+        <NavBar />
```
Comment (finnhodgkin) [inline src/containers/LikedCareers.js]: Again

File: src/containers/CareerDetails.js (hunk 2 above)
Comment (finnhodgkin) [reply src/containers/CareerDetails.js]: Also see the above code snippet for more user-friendly URLs.

Comment (matthewdking) [reply src/containers/CareerDetails.js]: So neat :sparkles:

**Pull request merged**